DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/ActivityDelegates/RemoveDelegateSelfAssessment.cshtml

```diff
@@ -0,0 +1,58 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates
+@using Microsoft.Extensions.Configuration
+@model DelegateSelfAssessmenteViewModel;
+@inject IConfiguration Configuration;
+@{
+    var errorHasOccurred = !ViewData.ModelState.IsValid;
+    ViewData["Title"] = (errorHasOccurred ? "Error: " : "") + "Delegate Self Assessment - Confirm Remove";
+}
+
+@section NavMenuItems {
+    <partial name="Shared/_NavMenuItems" />
+}
+
+<div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-full word-break">
+        @if (errorHasOccurred)
+        {
+            <vc:error-summary order-of-property-names="@(new[] { nameof(Model.ActionConfirmed) })" />
+        }
+        <h1 id="page-heading">Confirm Remove Delegate Self Assessment</h1>
+        <div class="nhsuk-grid-row">
+            <div class="nhsuk-grid-column-one-quarter nhsuk-heading-l">
+                <div class="nhsuk-u-font-weight-bold">
+                    Delegate self assessment:
+                </div>
+            </div>
+            <div class="nhsuk-grid-column-three-quarters nhsuk-heading-l nhsuk-u-font-weight-normal">
+                @Model.FirstName @Model.LastName (@Model.Email)
+            </div>
+        </div>
+
+        <form method="post" asp-controller="ActivityDelegates">
+            <div class="nhsuk-checkboxes__item">
+                <vc:single-checkbox asp-for="@nameof(Model.ActionConfirmed)"
+                                    label="I am sure that I wish to remove @Model.FirstName @Model.LastName from self assessment"
+                                    hint-text="This action will remove the delegate from your self assessment list and you will no longer be able to view this delegate from self assessments." />
+            </div>
+            <button type="submit" class="nhsuk-button nhsuk-button--danger nhsuk-u-margin-top-4" asp-action="RemoveDelegateSelfAssessment">
+                Remove
```

Comment:
Change button name to "Remove from self assessment"

Reply:
Done